src/Store/Reducer.re

```diff
@@ -52,14 +52,10 @@ let reduce: (State.t, Actions.t) => State.t =
       | ReallyQuitting => {...s, isQuitting: true}
       | WindowFocusGained => {...s, windowIsFocused: true}
       | WindowFocusLost => {...s, windowIsFocused: false}
-      | WindowMaximized => {...s, windowIsMaximized: true}
-      | WindowFullscreen => {...s, windowIsFullscreen: true}
+      | WindowMaximized => {...s, windowDisplayMode: Maximized}
```

Comment:
Thanks for moving from the correlated-bools to a variant, this seems much easier to work with than trying to understand the relationship between `isMaximized` and `isFullscreen` - now those states are mutually exclusive 👍